docs/Developer-Guide.md

```diff
@@ -210,7 +210,8 @@ $ sudo systemctl restart systemd-journald
 
 > **Note:**
 >
-> - You should only do this step if you are testing with the latest version of the agent.
+> - You should only do this step if you are testing with the latest version of the agent or you want to build
+>   a custom agent for seccomp.
```

Comment:
Is this line needed? SECCOMP is yes by default.

Reply:
Or change `for seccomp` to `without seccomp`?

Reply:
I adopted `a custom agent without seccomp feature`. Thank you.